styleSheet.css

```diff
@@ -29,22 +29,81 @@ ul {
   float: left;
 }
 
-.navBar__list>a{
+.navBar__list > a{
   color: white;
   display: block;
   padding: 14px 16px;
   text-align: center;
   text-decoration: none;
 }
 
-.navBar__list>a:hover{
+.navBar__list > a:hover{
   background-color: #111;
 }
 
-.navBar__list--active>a{
+.navBar__list--active > a{
   color: #4CAF50;
 }
+.navBar__list__closeBtn{
+  color: #818181;
+  display: none;
+  padding: 5px 5px 0px 0px;
+  text-align: right;
+  text-decoration: none;
+}
+
+#bodyResume {
+  position: relative;
+  top :5%;
+}
+.navBar__btn{
+  background-color: #111;
+  border: none;
+  color: #fff;
+  display: none;
+  font-size: calc(1em + 3.5vw);
+}
+
+.navBar__sidebarBtn{
+  height: 6.795%;
```

Comment:
this values look really odd? It is possible to use but is prefer to use integers